packages/debugger-extension/src/index.ts

```diff
@@ -638,6 +639,19 @@ const main: JupyterFrontEndPlugin<void> = {
           source: path
         });
         if (results.length > 0) {
+          if (breakpoint && typeof breakpoint.line === 'number') {
```

Comment:
Do we need the type check for `breakpoint.line`?

Reply:
Ah ok. Then maybe a check against `undefined`?

It looked a bit interesting at first because the type of `line` is `number` from the vscode type definitions.